community/cypher/acceptance/src/test/scala/org/neo4j/internal/cypher/acceptance/FunctionsAcceptanceTest.scala

```diff
@@ -219,6 +219,23 @@ class FunctionsAcceptanceTest extends ExecutionEngineFunSuite with NewPlannerTes
     result.toList should equal(List(Map("n" -> node)))
   }
 
+  test("percentileDisc should work in the valid range") {
+    createNode("prop" -> 10.0)
+    createNode("prop" -> 20.0)
+    createNode("prop" -> 30.0)
 
+    executeScalarWithAllPlanners[Double]("MATCH (n) RETURN percentileDisc(n.prop, 0.0)") should equal (10.0 +- 0.1)
+    executeScalarWithAllPlanners[Double]("MATCH (n) RETURN percentileDisc(n.prop, 0.5)") should equal (20.0 +- 0.1)
+    executeScalarWithAllPlanners[Double]("MATCH (n) RETURN percentileDisc(n.prop, 1.0)") should equal (30.0 +- 0.1)
+  }
+
+  test("percentileCont should work in the valid range") {
```

Comment:
What about negative acceptance tests? Ie, asserting that invalid values throw the expected error?